src/containers/Article.js

```diff
@@ -188,7 +188,14 @@ export default class Article extends Component {
             relateds={article.relateds}
           />
         </div>
-
+        <ArticleComponents.PageNavigation
+          article={ _.get(article, [ 'relateds', 0 ])}
+          navigate="next"
+        />
+        <ArticleComponents.PageNavigation
+          article={_.get(article, [ 'relateds', 1 ])}
+          navigate="previous"
+        />
```

Comment:
Do we need further discussion on how to let users go back to the previous article they've just clicked? Sry I just found NYT can let user go back while the first previous article might be selected from its backend